wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationReadReceiptSettingChangedCellDescription.swift

```diff
@@ -56,7 +61,7 @@ struct ReadReceiptViewModel {
         case .readReceiptsEnabled:
             updateText = createSystemMessage(template: "content.system.message_read_receipt_on")
         case .readReceiptsOn:
-            updateText = NSAttributedString(string: "content.system.message_read_receipt_on_add_to_group".localized, attributes: ConversationSystemMessageCell.baseAttributes)
+            updateText = NSAttributedString(string: "content.system.message_read_receipt_on_add_to_group".localized, attributes: baseAttributes)
```

Comment:
Can you please replace .localized with `L10n.Localizable...` We're trying to migrate over this type from `.localized`